solr/solrj/src/java/org/apache/solr/client/solrj/io/Tuple.java

```diff
@@ -226,7 +214,7 @@ public Double getDouble(Object key) {
   /**
    * Return all tuple fields and their values.
    */
-  public Map<Object, Object> getFields() {
+  public Map<String, Object> getFields() {
```

Comment:
As a result of this change two casts in `UpdateStream` could be removed, https://github.com/cpoerschke/solr/commit/d687763475aa43ee4e5fde652cdd05a581fe0fb9 illustrates.

Reply:
Yep, will incorporate this!